nx/include/switch/services/mii.h

```diff
@@ -133,25 +133,25 @@ Result miiOpenDatabase(MiiDatabase *out, MiiSpecialKeyCode key_code);
  * @brief Returns whether the mii database is updated.
  * @param[in] db Database.
  * @param[in] flag Source flag.
- * @param[out] out Out boolean.
+ * @param[out] out_updated Whether the mii database is updated.
  */
-Result miiDatabaseIsUpdated(MiiDatabase *db, u8 *out, MiiSourceFlag flag);
+Result miiDatabaseIsUpdated(MiiDatabase *db, bool *out_updated, MiiSourceFlag flag);
 
 /**
  * @brief Returns whether the mii database is full.
  * @param[in] db Database.
  * @param[in] flag Source flag.
- * @param[out] out Out boolean.
+ * @param[out] out_full Whether the mii database is updated.
```

Comment:
Duplicate comment.

Reply:
(was referring to this comment being identical to the one for miiDatabaseIsUpdated)

Reply:
Fixed the comment